Agents/FenlandTrajectoryAgent/docker-compose-debug.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.8'
+services:
+  fenland_trajectory_agent:
+    image: ghcr.io/cambridge-cares/fenland_trajectory_agentg:1.0.0-SNAPSHOT
```

Comment:
The name of the agent contains a typo! Please fix.

Also, don't forget to remove the snapshot identifier before merging.